plugins/wallabag.koplugin/main.lua

```diff
@@ -0,0 +1,692 @@
+local DocSettings = require("frontend/docsettings")
+local FFIUtil = require("ffi/util")
+local InfoMessage = require("ui/widget/infomessage")
+local LuaSettings = require("frontend/luasettings")
+local UIManager = require("ui/uimanager")
+local NetworkMgr = require("ui/network/manager")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local InputDialog = require("ui/widget/inputdialog")
+local MultiInputDialog = require("ui/widget/multiinputdialog")
+local logger = require("logger")
+local ltn12 = require('ltn12')
+local http = require('socket.http')
+local https = require('ssl.https')
+local socket = require('socket')
+local url = require('socket.url')
+local util = require("util")
+local _ = require("gettext")
+local T = FFIUtil.template
+local Screen = require("device").screen
+local DataStorage = require("datastorage")
```

Comment:
Please note the order in our coding standards.

Reply:
Done.